googleapis/google-cloud-node
# migrate code from googleapis/nodejs-data-qna

- chore: add initial files for launch
- chore: add initial files for launch
- feat!: initial library stub
- feat!: add samples to library (#1)
- chore: release 1.0.0 (#3)
- docs: add allowlist warning to README (#4)
- fix: fix docs links (#6)
- chore(deps): update dependency mocha to v8 (#7)
- chore: release 1.0.1 (#10)
- build: update synth.py, rerun synthtool (#13)
- build: update gapic-generator-typescript to v1.2.10. (#14)
- chore(deps): update dependency sinon to v10 (#16)
- chore: migrate to owl bot (#17)
- chore(deps): update dependency @types/sinon to v10 (#20)
- chore(deps): update dependency ts-loader to v9 (#24)
- fix(deps): require google-gax v2.12.0 (#28)
- chore: new owl bot post processor docker image (#30)
- fix: use require() to load JSON protos (#31)
- chore: update gapic-generator-typescript to v1.3.2 (#32)
- chore: release 1.0.2 (#29)
- chore(deps): update dependency sinon to v11 (#35)
- fix: GoogleAdsError missing using generator version after 1.3.0 (#36)
- chore: release 1.0.3 (#37)
- fix: make request optional in all cases (#43)
- chore: release 1.0.4 (#44)
- fix(deps): google-gax v2.17.0 with mTLS (#50)
- chore: release 1.0.5 (#51)
- fix(deps): google-gax v2.17.1 (#53)
- chore: release 1.0.6 (#54)
- fix: Updating WORKSPACE files to use the newest version of the Typescript generator. (#55)
- chore: release 1.0.7 (#56)
- chore(nodejs): update client ref docs link in metadata (#61)
- fix(build): migrate to using main branch (#63)
- fix(deps): google-gax v2.24.1 (#65)
- chore: release 1.0.8 (#64)
- feat: turns on self-signed JWT feature flag (#66)
- chore: release 1.1.0 (#67)
- build(node): update deps used during postprocessing (#1243) (#76)
- chore: fix the wrong post processor image (#78)
- chore(deps): update dependency @types/node to v16 (#77)
- chore(cloud-rad): delete api-extractor config (#79)
- chore(deps): update dependency sinon to v12 (#80)
- docs(samples): add example tags to generated samples (#81)
- build: add generated samples to .eslintignore (#82)
- docs(node): support "stable"/"preview" release level (#1312) (#85)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#86)
- chore: add api_shortname and library_type to repo metadata (#84)
- test(nodejs): remove 15 add 16 (#1322) (#88)
- chore: update v2.12.0 gapic-generator-typescript (#92)
- docs(nodejs): version support policy edits (#1346) (#96)
- chore(deps): update dependency sinon to v13 (#94)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#97)
- chore: update v2.14.2 gapic-generator-typescript (#101)
- build(node): update client library version in samples metadata (#1356) (#107)
- chore(deps): update dependency @types/mocha to v9 (#108)
- chore(deps): update dependency sinon to v14 (#111)
- build!: update library to use Node 12 (#112)
- chore(deps): update dependency jsdoc-region-tag to v2 (#117)
- chore(deps): update dependency jsdoc-fresh to v2 (#116)
- chore(main): release 2.0.0 (#114)
- fix(docs): document fallback rest option (#118)
- chore(deps): update dependency linkinator to v4 (#121)
- fix: better support for fallback mode (#122)
- fix: change import long to require (#123)
- fix: do not import the whole google-gax from proto JS (#1553) (#125)
- fix: allow passing gax instance to client constructor (#126)
- chore(main): release 2.0.1 (#119)
- fix: preserve default values in x-goog-request-params header (#127)
- test: use fully qualified request type name in tests (#129)
- fix(deps): use google-gax v3.5.2 (#133)
- chore(deps): update dependency @types/node to v18 (#131)
- chore(deps): update dependency jsdoc to v4 (#135)
- fix: regenerated protos JS and TS definitions (#136)
- chore(main): release 2.0.2 (#128)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
